PIL/Image.py

```diff
@@ -1814,6 +1893,7 @@ def transform(x, y, matrix):
         return self.transform((w, h), AFFINE, matrix, resample)
 
     def save(self, fp, format=None, **params):
+        # type: (Union[Text, pathlib.Path, BytesIO], Optional[Text], **Any) -> None
```

Comment:
StringIO.StringIO is binary capable in py2 (and removed in py3), but it was replaced with the io.*Io classes. It's the file to write to, and in open, read from. But I have no idea how that's going to interact with mypy, as it's only defined in py2.  This may be another case where it's unlikely that people who are using typing are going to be using StringIO, as they're going to be modernizing other sections of their code, and io.BytesIO is a drop in replacement.